src/code42cli/cmds/users.py

```diff
@@ -141,6 +148,17 @@ def update_user(
     "archive_size_quota",
 ]
 
+_bulk_user_move_headers = ["username", "org_id"]
+
+
+@users.command(name="move")
+@username_option("The username of the user to be moved.", required=True)
+@org_id_option
+@sdk_options()
+def change_organization(state, username, org_id):
```

Comment:
Would org name be a more friendly input param?

Reply:
The difficulty with org name is that we don't have a good way to get the org ID from the org name, we'd have to do `sdk.orgs.get_all` and then iterate through the result to find what we want. Org names are also not guaranteed to be unique even within a tenant (they should be, but there are a few different ways to end up with non-unique org names).